### PR TITLE
Add release blog post for v1.19.0

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2024-01-29-new-ferretdb-v119-release.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+image: /img/blog/ferretdb-v1.19.0.jpg
+tags: [release]
+---
+
+![FerretDB releases v1.19.0](/img/blog/ferretdb-v1.19.0.jpg)
+
+We just released FerretDB v1.19.0, which includes support for creating an index on nested fields in SQLite and some important bug fixes.
+
+In recent weeks, we've been working on enabling support for authentication.
+That way, [FerretDB](https://www.ferretdb.com/) can manage users by itself and not rely on the backend to do so.
```
Comment (AlekSi) [inline website/blog/2024-01-29-new-ferretdb-v119-release.md:16]: Authentication is already exist and enabled: https://docs.ferretdb.io/security/authentication/

File: website/blog/2024-01-29-new-ferretdb-v119-release.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Our goal is to make FerretDB the truly open source MongoDB alternative, and performance is also a key part of that.
+We have been focusing on improving the performance of FerretDB, and this might be noticeable already in some of the latest releases.
```
Comment (AlekSi) [inline website/blog/2024-01-29-new-ferretdb-v119-release.md:19]: It is not noticeable yet

File: website/blog/2024-01-29-new-ferretdb-v119-release.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+We also fixed an issue with search queries on the documentation.
+Before, our documentation search wasn't able to handle queries that involved operators like `$eq`, and this is finally fixed.
+
+## Other changes
+
+Many of the recent changes have gone into enabling support for authentication on FerretDB.
+To make that possible, we've recently added support for more user management commands, such as the `updateUser` command.
+
+We hope to have support for authentication available in future releases.
```
Comment (AlekSi) [inline website/blog/2024-01-29-new-ferretdb-v119-release.md:55]: That's very wrong

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: website/blog/2024-01-30-new-ferretdb-v119-release.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+We also fixed an issue with search queries on the documentation.
+Before, our documentation search wasn't able to handle queries that involved operators like `$eq`, and this is finally fixed.
+
+## Other changes
+
+Many of the changes in recent releases focus on enabling support for new authentication mechanism, such as `SCRAM-256`.
+Among other things needed to make this possible, FerretDB should manage users by itself and support more user management commands.
+So far, we've added basic support for the following user management commands: `createUser`, `dropAllUsersFromDatabase`, `dropUser`, `updateUser`, and `usersInfo`.
+
+For now, they are only accessible for testing purposes by running FerretDB with the `EnableNewAuth` flag/`FERRETDB_TEST_ENABLE_NEW_AUTH` environment variable.
```
Comment (Fashander) [inline website/blog/2024-01-30-new-ferretdb-v119-release.md:57]: @AlekSi I'm not sure if this is something we want to include. Also, while I feel this should work -`EnableNewAuth` flag didn't work for me when I tried it out, but the environment variable `FERRETDB_TEST_ENABLE_NEW_AUTH` did.

Comment (AlekSi) [reply website/blog/2024-01-30-new-ferretdb-v119-release.md:57]: Where did you get that flag name from?

File: website/blog/2024-01-30-new-ferretdb-v119-release.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+In [our last release (v1.18.0)](https://blog.ferretdb.io/new-ferretdb-v118-support-oplog-functionality/), we added support for OpLog functionality, and we documented its usage and how you can set it up for your application.
+With the addition of OpLog, users can start building real-time applications with FerretDB using the Meteor framework – [See the docs here](https://docs.ferretdb.io/configuration/oplog-support/).
+
+We also fixed an issue with search queries on the documentation.
+Before, our documentation search wasn't able to handle queries that involved operators like `$eq`, and this is finally fixed.
+
+## Other changes
+
+Many of the changes in recent releases focus on enabling support for new authentication mechanism, such as `SCRAM-256`.
```
Comment (AlekSi) [inline website/blog/2024-01-30-new-ferretdb-v119-release.md:53]:
```suggestion
Many of the changes in recent releases focus on enabling support for new authentication mechanisms, such as `SCRAM-SHA-256`.
```

File: website/blog/2024-01-30-new-ferretdb-v119-release.md (hunk 4 above)
Comment (AlekSi) [reply website/blog/2024-01-30-new-ferretdb-v119-release.md:57]: Let's keep it, with the right flag name

**Review by AlekSi — APPROVED** [review]

**Pull request merged**